src/logic/utils.ts

```diff
@@ -41,13 +56,18 @@ export function getAverageWinTurn(player: Player) {
     return average.toFixed(1);
 }
 
+/**
+ * Used to define what highlights a newly qualified player.
```

Comment:
nit: here's a format for getter that I like to use.

Given NNN, returns YYY.

So in this case,

Given a player, returns if that player is considered "newly qualified" based on a certain number of days since their last match and the number of matches they have played.

Reply:
used the following wording:
Given a player returns whether they're considered to be a newcomer on the stats or not. Counts valid matches only.

Reply:
in this case, you are introducing a new concept of "newcomer" to the language of our documentation. I would prefer that we stick to "newly qualified player". You can also drop the "on the stats or not" part because this method may not be strictly called on a stats page, it could be called within a test, etc.

Reply:
done & fix'd